### PR TITLE
[Docs]  Add Chinese version of customize_models

### PR DESCRIPTION
**Pull request by jliu-ac**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Notes

The translation of some specific professional terms needs to be confirmed. Listening to your suggestions and I will follow up and keep maintaining it.

## Motivation

Add Chinese version of customize_models

## Modification

Add Chinese version of customize_models

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs_zh-CN/tutorials/customize_models.md [hunk 1]
```diff
@@ -1 +1,369 @@
 # 教程 4: 自定义模型
+
+我们基本可以把模型的各个组件分为五类：
+
+- 主干网络 (backbone)：通常是一个用来提取特征图 (feature map) 的全连接网络 (FCN network)，例如：ResNet, MobileNet。
+- 颈部 (neck)：主干网络和头部之间的部分，例如：FPN, PAFPN。
+- 头部 (head)：用于具体任务的组件，例如：边界框预测和掩膜预测。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_models.md:7]: 掩膜 ->  掩码

File: docs_zh-CN/tutorials/customize_models.md [hunk 2]
```diff
@@ -1 +1,369 @@
...
+- 区域提取器 (roi extractor)：从特征图中提取RoI特征，例如：RoI Align。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_models.md:8]: 提取 RoI 特征
Need space between Chinese and English

File: docs_zh-CN/tutorials/customize_models.md [hunk 3]
```diff
@@ -1 +1,369 @@
...
+到配置文件来实现相同的目的。
+
+如使用，请修改 `loss_xxx` 字段。
+因为 MyLoss 是用于回归的，你需要在头部中修改 `loss_xxx` 字段。
+
+```python
+loss_bbox=dict(type='MyLoss', loss_weight=1.0))
+```
+
+
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_models.md:358]: Delete extra blank lines

File: docs_zh-CN/tutorials/customize_models.md [hunk 4]
```diff
@@ -1 +1,369 @@
 # 教程 4: 自定义模型
+
+我们基本可以把模型的各个组件分为五类：
+
+- 主干网络 (backbone)：通常是一个用来提取特征图 (feature map) 的全连接网络 (FCN network)，例如：ResNet, MobileNet。
+- 颈部 (neck)：主干网络和头部之间的部分，例如：FPN, PAFPN。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_models.md:6]: Neck, Head 可以不翻译

File: docs_zh-CN/tutorials/customize_models.md [hunk 5]
```diff
@@ -1 +1,369 @@
...
+    in_channels=[256, 512, 1024, 2048],
+    out_channels=256,
+    num_outs=5)
+```
+
+### 添加新的头部
+
+这里，我们以 [Double Head R-CNN](https://arxiv.org/abs/1904.06493) 为例来展示如何开发一个新的头部。
+
+首先，添加一个新的边界框头（bbox head）到 `mmdet/models/roi_heads/bbox_heads/double_bbox_head.py`。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_models.md:123]: bbox head 可以不翻译

**Comment by BIGWangYuDong** [comment]
You can follow https://github.com/open-mmlab/mmcv/blob/9d4571e3d0bddb7e49852909eddcf5b4145c1a91/docs_zh_CN/community/contributing.md to prepare  pre-commit hook

File: docs_zh-CN/tutorials/customize_models.md [hunk 6]
```diff
@@ -1 +1,356 @@
 # 教程 4: 自定义模型
+
+我们简单地把模型的各个组件分为五类：
+
+- 主干网络 (backbone)：通常是一个用来提取特征图 (feature map) 的全连接网络 (FCN network)，例如：ResNet, MobileNet。
```
Comment (hhaAndroid) [inline docs_zh-CN/tutorials/customize_models.md:5]: 全连接 -> 全卷积

File: docs_zh-CN/tutorials/customize_models.md [hunk 7]
```diff
@@ -1 +1,356 @@
...
+- Neck：主干网络和 Head 之间的部分，例如：FPN, PAFPN。
```
Comment (hhaAndroid) [inline docs_zh-CN/tutorials/customize_models.md:6]: 主干网络和 Head 之间的部分 -> 主干网络和 Head 之间的连接部分

File: docs_zh-CN/tutorials/customize_models.md [hunk 8]
```diff
@@ -1 +1,356 @@
...
+    in_channels=[256, 512, 1024, 2048],
+    out_channels=256,
+    num_outs=5)
+```
+
+### 添加新的 Head
+
+这里，我们以 [Double Head R-CNN](https://arxiv.org/abs/1904.06493) 为例来展示如何开发一个新的 Head。
+
+首先，添加一个新的 bbox head 到 `mmdet/models/roi_heads/bbox_heads/double_bbox_head.py`。
```
Comment (hhaAndroid) [inline docs_zh-CN/tutorials/customize_models.md:123]: This passage is not coherent and may have to be revised.

**Review by BIGWangYuDong — APPROVED** [review]
LGTM

**Review by RangiLyu — APPROVED** [review]
LGTM

**Review by ZwwWayne — APPROVED** [review]

**Comment by ZwwWayne** [comment]
Hi @jliu-ac ，
We have refactored the documentation structure. Would you like to resolve the conflict so that we could merge this PR?

**Comment by jliu-ac** [comment]
> Hi @jliu-ac ，
> We have refactored the documentation structure. Would you like to resolve the conflict so that we could merge this PR?

I think I would like to resolve the conflict, so I would like to know the way to resolve. This is my first time to make contribution to such a big project, really want to get some guide about it.

**Comment by BIGWangYuDong** [comment]
Hi, you can first change to the master branch and pull the master. Then you can change to cn_doc branch and merge the master branch

**Comment by RangiLyu** [comment]
Kindly ping @ZwwWayne

**Comment by OpenMMLab-Assistant003** [comment]
Hi @jliu-ac！First of all, we want to express our gratitude for your significant PR in the MMDetection project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤ @jliu-ac

**Pull request merged**